great-idea/index.html

```diff
@@ -15,9 +15,102 @@
 </head>
 
   <body>
-
-    <!-- Copy and paste your HTML from the first UI project here -->
-      
+    <!-- Main Cointainer-->
+    <div id="main_container">
+  <!--Navigation Container-->
+  <div id="nav">
+    <nav>
+        <a href="services.html">Services</a> 
+        <a href="/css/">Product</a> 
+        <a href="vision.html">Vision</a> 
+        <a href="/jquery/">Features</a>
+        <a href="/html/">About</a> 
+        <a href="/css/">Contact</a>
+    </nav>
+    <img class="logo" src="img/logo.png" alt="Great Idea! Company logo.">
+  </div>
+  <!--Heading Container-->
+  <div id="heading">
+    <!--Heading Text Container-->
+    <div id="heading_text">
+      <h1>
+        Innovation</br>
+        On</br>
+        Demand
+       </h1>
+       <button type="button">Get Started</button>   
+     </div>
+     <!--Heading Image Container--> 
+     <div id="heading_image">
+      <img src="img/header-img.png" alt="Image of a code snippet.">
+     </div>
+  </div> 
+  <!--Page Break-->
+  <hr >
+  <!--Content Container-->
+  <div id="content_container">
+   
+    <!--Features Container-->
+    <div class="top_content">
+      <h2>Features</h2>
+      <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+    </div>
+    
+    <!--About Container-->
+    <div class="top_content">
+      <h2>About</h2>
+      <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+    </div>
+    
+    <!--Image Container-->
```

Comment:
I like seeing how you are starting to add lots of comments to yourself so that you know exactly what is going on - keep that up!